source/class/qx/tool/compiler/targets/Target.js

```diff
@@ -395,12 +396,15 @@ module.exports = qx.Class.define("qx.tool.compiler.targets.Target", {
             var arr = library.getAddScript();
```

Comment:
IMHO this should be done for the Css Array to. They should also be enabled to use externals.